python/ray/worker.py

```diff
@@ -2188,6 +2190,8 @@ def method(self):
         retry_exceptions (bool): Only for *remote functions*. This specifies
             whether application-level errors should be retried
             up to max_retries times.
+        scheduling_policy (str): Policy about how to schedule
```

Comment:
@ericl @rkooo567 Should we call it scheduling_strategy to be more coherent with placement group strategy?

Reply:
Yeah +1. (Also using the same term for values)